routes/login.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+var nconf = require('nconf');
+var express = require('express');
+var bodyParser = require('body-parser')
+var router = express.Router();
+
+// create application/json parser
+var jsonParser = bodyParser.json()
+// create application/x-www-form-urlencoded parser
+var urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+var redirectBack = function(req, res) {
+    var redirectAfterLogin = req.session.redirectAfterLogin ? req.session.redirectAfterLogin : '/';
```

Comment:
Shorthand for this is `var redirectAfterLogin = req.session.redirectAfterLogin || '/'`.  In JS, `||` acts as a "fallback" command.